{"name": "test_delete_private_repository", "status": "failed", "statusDetails": {"message": "AssertionError: Private repo deletion failed or already deleted\nassert 401 in [204, 404]\n +  where 401 = <Response [401]>.status_code", "trace": "def test_delete_private_repository():\n        \"\"\"Step 7: Delete the private repository after validation.\"\"\"\n        print(\"\\nðŸ”¹ Running: test_delete_private_repository\")\n        response = requests.delete(f\"{GITHUB_API_URL}/repos/{OWNER}/{PRIVATE_REPO_NAME}\", headers=HEADERS)\n>       assert response.status_code in [204, 404], \"Private repo deletion failed or already deleted\"\nE       AssertionError: Private repo deletion failed or already deleted\nE       assert 401 in [204, 404]\nE        +  where 401 = <Response [401]>.status_code\n\ntests\\api_tests\\test_authentication_api.py:87: AssertionError"}, "description": "Step 7: Delete the private repository after validation.", "attachments": [{"name": "stdout", "source": "3b3b8d2c-5a8e-42cb-823d-b343760c98ca-attachment.txt", "type": "text/plain"}], "start": 1742175258680, "stop": 1742175258959, "uuid": "fe1b219c-26df-43ac-bcf2-dc0adce84fbc", "historyId": "8de8b9626359fa252a8906205f3e3d53", "testCaseId": "8de8b9626359fa252a8906205f3e3d53", "fullName": "tests.api_tests.test_authentication_api#test_delete_private_repository", "labels": [{"name": "parentSuite", "value": "tests.api_tests"}, {"name": "suite", "value": "test_authentication_api"}, {"name": "host", "value": "BHUVANSPC"}, {"name": "thread", "value": "9344-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api_tests.test_authentication_api"}]}